### PR TITLE
Add cigetcert

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This adds a recipe for cigetcert.

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/cigetcert/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: db7c3226552d23516aa16a767dfce8031362640fc7a94d87a0a6e90d5c8afb36
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (xhochy) [inline recipes/cigetcert/meta.yaml]: These are no longer needed but applied as default
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
```

File: recipes/cigetcert/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: db7c3226552d23516aa16a767dfce8031362640fc7a94d87a0a6e90d5c8afb36
+
+build:
+  number: 0
```
Comment (xhochy) [inline recipes/cigetcert/meta.yaml]: Can this be `noarch: python`?

Comment (duncanmmacleod) [reply recipes/cigetcert/meta.yaml]: @xhochy, no, because it uses `scripts` in `setup.py`.

Comment (duncanmmacleod) [reply recipes/cigetcert/meta.yaml]: Also, as it turns out, `pykerberos` isn't available on windows, so I need to skip that for now, and will work with upstream to try and support windows in the future.

**Comment by duncanmmacleod** [comment]
@conda-forge/help-python, I think this recipe is ready for review, thanks!

**Comment by duncanmmacleod** [comment]
Thanks @beckermr, @xhochy!

**Pull request merged**